packages/design-system/package.json

```diff
@@ -0,0 +1,153 @@
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/prop-types": "^15.5.6",
+    "@types/react": "^16.4.15",
+    "@types/react-dom": "^16.0.8",
+    "autoprefixer": "^9.0.1",
+    "babel-cli": "^6.26.0",
+    "babel-eslint": "^8.2.6",
+    "babel-jest": "^23.4.2",
+    "babel-loader": "^7.1.1",
+    "babel-plugin-css-modules-transform": "^1.6.1",
+    "babel-plugin-external-helpers": "^6.22.0",
+    "babel-plugin-react-svg": "^2.1.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-imports": "^1.5.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-plugin-transform-react-jsx": "^6.24.1",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.15",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-es2015-rollup": "^3.0.0",
+    "babel-preset-react": "^6.3.13",
+    "caniuse-lite": "1.0.30000974",
+    "cross-env": "^5.2.0",
+    "css-loader": "^2.1.1",
+    "cssnano": "^4.0.5",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "enzyme-to-json": "^3.3.4",
+    "eslint": "^4.19.1",
+    "eslint-config-airbnb": "^16.1.0",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-import-resolver-webpack": "^0.10.0",
+    "eslint-plugin-babel": "^5.1.0",
+    "eslint-plugin-import": "^2.12.0",
+    "eslint-plugin-jsx-a11y": "^6.0.3",
+    "eslint-plugin-prettier": "^2.6.2",
+    "eslint-plugin-react": "^7.8.2",
+    "eslint-plugin-typescript": "^0.12.0",
+    "extract-text-webpack-plugin": "^3.0.2",
+    "gh-pages": "^1.2.0",
+    "husky": "^1.3.1",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^23.4.2",
+    "lint-staged": "^8.1.5",
+    "mini-css-extract-plugin": "^0.5.0",
+    "node-eval": "^2.0.0",
+    "node-sass": "^4.9.3",
+    "normalize.css": "^8.0.0",
+    "postcss-flexbugs-fixes": "^4.0.0",
+    "postcss-global-import": "^1.0.6",
+    "postcss-import": "^11.1.0",
+    "postcss-load-config": "^2.0.0",
+    "postcss-loader": "^2.1.6",
+    "postcss-modules": "^1.3.2",
+    "postcss-url": "^7.3.2",
+    "precss": "^3.1.2",
+    "prettier": "1.14.2",
+    "prettier-eslint-cli": "^4.7.0",
+    "prettier-stylelint": "^0.4.2",
+    "react": "^16.3.1",
+    "react-dom": "^16.3.1",
+    "react-test-renderer": "^16.8.4",
+    "rimraf": "^2.6.1",
+    "rollup": "^0.67.3",
+    "rollup-plugin-babel": "^3.0.7",
+    "rollup-plugin-commonjs": "^8.2.1",
+    "rollup-plugin-node-resolve": "^3.0.0",
+    "rollup-plugin-postcss": "^1.6.2",
+    "rollup-plugin-replace": "^1.1.1",
+    "rollup-plugin-typescript2": "^0.17.1",
+    "rollup-plugin-uglify": "^2.0.1",
+    "sass-loader": "^7.1.0",
+    "style-loader": "^0.23.1",
+    "stylelint": "^9.6.0",
+    "stylelint-config-recommended-scss": "^3.2.0",
+    "stylelint-config-standard": "^18.2.0",
+    "stylelint-scss": "^3.3.1",
+    "ts-loader": "^3.5.0",
+    "typescript": "^2.9.1",
+    "typescript-eslint-parser": "^19.0.2",
+    "webpack": "^4.28.4",
```

Comment:
this package right now doesnt depend anyhow on webpack, so it should be removed from here (along with any other webpack-related devDeps)

Reply:
done at 538a33315631085f7d995318f66c3a90ddf9c327

Reply:
done at 538a33315631085f7d995318f66c3a90ddf9c327